### DISCUSSION TITLE
Add custom background to webui?

### DISCUSSION THREAD

**Discussion by scgalvin** [oobabooga/text-generation-webui#2325] [Q&A]
How would I add a custom background to the webui? something similar to the Tavern AI feature?

**Comment by m-spangenberg** [comment]
You would create Gradio elements for the web-ui that allows you to point to an image and then write some CSS that gets processed by Gradio, like so: [How to add background to Gradio](https://discuss.huggingface.co/t/add-background-image/16381).

**Comment by flflow** [comment]
Add css rule to one of css files or via browser extension like "Stylus" and place background.jpg to text-generation-webui folder.
.gradio-container > .main {background-image: url(/file/background.jpg); background-size: 100%; }

![2023-06-02 21 55 40](https://github.com/oobabooga/text-generation-webui/assets/32948829/9299b6d5-d13b-4639-8338-c83f24d185ec)
![2023-06-02 21 55 53](https://github.com/oobabooga/text-generation-webui/assets/32948829/49b03afb-fd50-42eb-b280-c1ac4539ae54)
![Uploading 2023-06-02 21 55 53.png…]()
